solvers/relay_constellation/umcf_srr_contact_plan/src/solve.py

```
@@ -0,0 +1,442 @@
+
+
+def _load_srr_config(config_dir: str | Path | None) -> SRRConfig:
+    """Load SRR config from config_dir/config.yaml if present."""
+    if not config_dir:
+        return SRRConfig()
+    config_path = Path(config_dir) / "config.yaml"
+    if not config_path.is_file():
+        return SRRConfig()
+    try:
+        raw = yaml.safe_load(config_path.read_text(encoding="utf-8")) or {}
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)